config.go

```diff
@@ -80,6 +80,8 @@ type ClientConfig struct {
 // APIConfig holds relevant log and listener settings for the API server.
 type APIConfig struct {
 	HttpListenAddr string
+	// TODO: where is the right place for this?
+	GithubWebhookSecret string `yaml:"github_webhook_secret"`
```

Comment:
There will be other data we will add here (authorized repos), so we may end up moving it, but this is fine for now.

Reply:
Authorized organizations, not repos.